src/shared/reducers/message.js

```diff
@@ -16,10 +16,24 @@ import {
   FETCH_FAILURE,
 } from "../actions/constants";
 
-export const addErrorToState = (state, { error }) => ({
-  ...state,
-  message: error,
-});
+export const addErrorToState = (state, { error }) => {
+  let message = error;
+
+  if (message instanceof Error) {
+    ({ message } = error);
+  }
+
+  if (typeof message !== "string") {
+    return {
```

Comment:
`return state;`?

Reply:
also I would log an error here because it should not happen.

Reply:
Is it allow to return the same state ? I thought you must return a new instance.

Reply:
if you dont change it, there is nothing else to do.